src/sentry/backup/imports.py

```diff
@@ -144,12 +144,21 @@ def yield_json_models(src) -> Iterator[Tuple[NormalizedModelName, str]]:
         if last_seen_model_name is not None and batch:
             yield (last_seen_model_name, json.dumps(batch))
 
+    def generate_organization_slug_reservations(org_ids: Set[Tuple[int, str]]):
+        from sentry.services.organization import organization_provisioning_service
+
+        for org_id, org_slug in org_ids:
+            organization_provisioning_service.change_organization_slug(
```

Comment:
Is there any way to do this in parallel? If I've imported 50 orgs, it could be quite slow to synchronously wait for 50 RPCs to round trip in sequence, rather than all at once.

Reply:
Addressed via a bulk import. It may still be partially synchronous, but it cuts down the number of network requests significantly

Reply:
I thought we were going to use the bulk creation method? Or are we no longer doing that here?